test/dynamo/test_misc.py

```diff
@@ -7858,6 +7858,54 @@ def my_dyn_fn(x, y):
         with self.assertRaises(ConstraintViolationError):
             torch.compile(my_dyn_fn, backend="eager")(y, y)
 
+    @torch.compiler.config.patch(dynamic_sources="L['x']")
+    def test_dynamic_int_sources(self):
```

Comment:
nit: if you same same prefix for all we can test them and find them in one command 
test_dynamic_sources_int/tesnor..etc
then I can do -k test_dynamic_sources